src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/InTableFilter.java

```diff
@@ -0,0 +1,104 @@
+package io.stargate.sgv2.jsonapi.service.operation.filters.table;
+
+import static com.datastax.oss.driver.api.querybuilder.QueryBuilder.bindMarker;
+
+import com.datastax.oss.driver.api.core.metadata.schema.ColumnMetadata;
+import com.datastax.oss.driver.api.querybuilder.relation.OngoingWhereClause;
+import com.datastax.oss.driver.api.querybuilder.relation.Relation;
+import com.datastax.oss.driver.api.querybuilder.term.Term;
+import io.stargate.sgv2.jsonapi.exception.ErrorCodeV1;
+import io.stargate.sgv2.jsonapi.exception.catchable.MissingJSONCodecException;
+import io.stargate.sgv2.jsonapi.exception.catchable.ToCQLCodecException;
+import io.stargate.sgv2.jsonapi.exception.catchable.UnknownColumnException;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.TableSchemaObject;
+import io.stargate.sgv2.jsonapi.service.operation.builder.BuiltCondition;
+import io.stargate.sgv2.jsonapi.service.operation.filters.table.codecs.JSONCodecRegistries;
+import io.stargate.sgv2.jsonapi.service.operation.query.TableFilter;
+import io.stargate.sgv2.jsonapi.service.operation.query.TableFilterAnalyzedUsage;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class InTableFilter extends TableFilter {
+
+  private final List<Object> arrayValue;
+
+  private final Operator operator;
+
+  public enum Operator {
+    IN,
```

Comment:
Is there expectation that `NIN` will be added? If not, is there any need for `Operator` enumeration, field?

Reply:
yes, will be added. [Ticket](https://github.com/stargate/data-api/issues/1478)